client/checks.go

```diff
@@ -172,3 +172,38 @@ func (client *Client) doMultiCheckAction(actionName string, checks []string) (*C
 
 	return results, nil
 }
+
+type RefreshCheckOptions struct {
+	// Name of the check to refresh, required.
```

Comment:
Nit for style and consistency:
```suggestion
	// Name of the check to refresh (required).
```